zbus/src/connection/builder.rs

```diff
@@ -80,26 +80,6 @@ pub struct Builder<'a> {
 
 assert_impl_all!(Builder<'_>: Send, Sync, Unpin);
 
-impl fmt::Debug for Builder<'_> {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let mut f = f.debug_struct("Builder");
-        f.field("target", &self.target);
-        f.field("max_queued", &self.max_queued);
-        #[cfg(feature = "p2p")]
-        f.field("guid", &self.target);
-        #[cfg(feature = "p2p")]
-        f.field("p2p", &self.p2p);
-        f.field("internal_executor", &self.internal_executor);
-        f.field("names", &self.names);
-        f.field("auth_mechanisms", &self.auth_mechanisms);
-        #[cfg(feature = "bus-impl")]
-        f.field("unique_name", &self.unique_name);
-        f.field("cookie_context", &self.cookie_context);
-        f.field("cookie_id", &self.cookie_id);
-        f.finish_non_exhaustive()
-    }
-}
-
```

Comment:
If you move the `Interface: Debug` commit at the top (first one), you'll not need to remove this code you're adding in a previous commit in the same PR.

Reply:
I know, but:
- it is additional work
- the way it is, `Interface: Debug` and `remove custom Debug ...` commits are revertible. If you decide this was a bad idea, you can go back and have the manual impls.

Reply:
> * it is additional work

Unfortunately, doing things right, typically is more work.

> * the way it is, `Interface: Debug` and `remove custom Debug ...` commits are revertible. If you decide this was a bad idea, you can go back and have the manual impls.

Ok, now that's a good argument. Let's keep it this way then. :)

Reply:
> Unfortunately, doing things right, typically is more work.

Not in this case, it seems.

Reply:
> > Unfortunately, doing things right, typically is more work.
> 
> Not in this case, it seems.

"typically" being the keyword here. :) My point was just that "more work" is not a good argument on its own. But you've a better argument so it doesn't matter, here. :)